File: ChickInRun/Assets/Script/ElectricColor.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class ElectricColor : MonoBehaviour
{
    public Material darkMatA;
    public Material darkMatB;
    public Material originMat;

    // Start is called before the first frame update
    void Start()
    {

    }

    // Update is called once per frame
    void Update()
    {
       
    }

    /// <summary>
    /// 2019
    /// </summary>
    /// <param name="num"></param>
    void ElecChange(int num)
    {
        if (num == 1)
        {
            GetComponent<Renderer>().material = darkMatA;
        }
        else if (num == 2)
        {
            GetComponent<Renderer>().material = darkMatB;
        }
        else if (num == 0)
        {
            GetComponent<Renderer>().material = originMat;
        }
    }
}

File: ChickInRun/Assets/Script/ElectricColor.js
#pragma strict

private var electDeath : GameObject;
private var elecVar : int;
public var darkMatA : Material;
public var darkMatB : Material;
public var originMat : Material;

function Start () {
	electDeath = GameObject.FindGameObjectWithTag("Chick");
	//originMat = renderer.material;
}

function Update () {
	elecVar = electDeath.GetComponent(PlayerInst).electricVar;
	if(elecVar == 1){
		GetComponent.<Renderer>().material = darkMatA;
	}
	else if(elecVar == 2){
		GetComponent.<Renderer>().material = darkMatB;
	}
	else if(elecVar ==0){
	
		GetComponent.<Renderer>().material = originMat;
	}
}